frontend/packages/ux-editor/src/utils/component.test.ts

```diff
@@ -237,8 +242,18 @@ describe('Component utils', () => {
     it('Returns array of unsupported property keys when known unsupported property keys are provided', () => {
       const properties = {
         children: 'testValue',
+        testProperty1: {
+          type: 'object',
+          properties: {},
+          additionalProperties: {
+            type: 'string',
+          },
+        },
       };
-      expect(getUnsupportedPropertyTypes(properties, ['children'])).toEqual(['children']);
+      expect(getUnsupportedPropertyTypes(properties, ['children'])).toEqual([
+        'testProperty1',
+        'children',
+      ]);
```

Comment:
Not directly connected to this PR, but isn't `children` supported by the treeview? Should it be listed here then?

Reply:
This should be fixed now!